src/shogun/base/SGObject.h

```diff
@@ -542,34 +474,7 @@ class CSGObject
 	 * @index index of the parameter
 	 * @return object parameter
 	 */
-	CSGObject* get(const std::string& name, index_t index) const;
-
-#ifndef SWIG
-	/** Typed setter for an object class parameter of a Shogun base class type,
-	 * identified by a name.
-	 *
-	 * @param name name of the parameter
-	 * @param value value of the parameter
-	 */
-	template <class T, class = typename std::enable_if_t<is_sg_base<T>::value>>
-	void put(const std::string& name, Some<T> value)
-	{
-		put(name, value.get());
-	}
-
-	/** Typed appender for an object class parameter of a Shogun base class
-	* type,
-	* identified by a name.
-	*
-	* @param name name of the parameter
-	* @param value value of the parameter
-	*/
-	template <class T, class = typename std::enable_if_t<is_sg_base<T>::value>>
-	void add(const std::string& name, Some<T> value)
```

Comment:
What about those? No more add?

Reply:
mmm i think that's one of the accidental deletion of rebase... :S will look into it as soon as i've got other blockers fixed thnx for reporting

Reply:
It’s tested in meta examples btw.... mkl and put-get-add